src/org/labkey/test/pages/ManageDatasetsPage.java

```diff
@@ -16,7 +16,8 @@
 package org.labkey.test.pages;
```

Comment:
Move to appropriate package:
```suggestion
package org.labkey.test.pages.study;
```